### DISCUSSION TITLE
Add BigQuery to Mongo operator at providers.mongo.transfers

### DISCUSSION THREAD

**Discussion by Wuerike** [apache/airflow#30791] [Ideas]
### Description

I've developed a BigQuery to MongoDB operator and I was thinking about contributing on adding it to the project.

I've basically merged BigQueryGetDataOperator (providers.google.cloud.operators) with MongoToS3Operator (providers.amazon.aws.transfers), following the same code style.

Has features like:
- Fetchs all fields on a line, or can receive witch fields should be fetch
- Stores the last fecthed line to read incrementaly between execution
- Supports this mongo methods: inserting, replacing and upserting 

I'd like to know if its good enough, and if someone is available to help me during the process of contributing.

### Use case/motivation

First of all, I'd like to make my first contribution on an open source project ever.

Talking about the application itself, it lacks of mongo operators, so I think it will be great to the community having one more example and application with it.

BigQuery already has a lot of operators this is why my idea was to add this on mongo folder.

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by potiuk** [comment]
Go for it! No  need to evan open an issue for it - but feel free. I assign you to it.

**Comment by eladkal** [comment]
@Wuerike  are you still working on this issue?

**Comment by Wuerike** [comment]
My idea was to do it, but I've forgotten about it actually
